zio-schema/jvm/src/main/scala/zio/schema/SchemaPlatformSpecific.scala

```diff
@@ -6,7 +6,7 @@ trait SchemaPlatformSpecific {
     Schema[String].transformOrFail(
       string =>
         try {
-          Right(new java.net.URL(string))
+          Right(new java.net.URI(string).toURL)
```

Comment:
I sneaked that one in. The URL constructor is deprecated in Java 21+ and this is the recommended migration